examples/svm/plot_rbf_parameters.py

```diff
@@ -171,11 +171,8 @@ def __call__(self, value, clip=None):
     plt.yticks(())
     plt.axis('tight')
 
-# plot the scores of the grid
-# grid_scores_ contains parameter settings and scores
-# We extract just the scores
-scores = [x[1] for x in grid.grid_scores_]
-scores = np.array(scores).reshape(len(C_range), len(gamma_range))
+scores = grid.results_['test_accuracy_mean'].reshape(len(C_range),
```

Comment:
It bugged me too but I couldn't think of a way. Do you have any suggestions in mind?